qutebrowser/browser/webkit/network/filescheme.py

```diff
@@ -124,7 +124,7 @@ def handler(request, _operation, _current_url):
     """
     path = request.url().toLocalFile()
     try:
-        if os.path.isdir(path):
+        if pathlib.Path(path).is_dir():
```

Comment:
I'd do `path = pathlib.Path(request.url().toLocalFile())` instead. We should be using `Path` to represent path variables wherever they are created, instead of passing around strings and just converting them to `Path` objects on-site